src/readsbml.jl

```diff
@@ -71,6 +71,34 @@ function get_optional_double(x::VPtr, is_sym, get_sym)::Maybe{Float64}
     end
 end
 
+function _readSBML(
+    symbol::Symbol,
+    fn::String,
+    sbml_conversion = document -> nothing,
+    report_severities = ["Fatal", "Error"],
```

Comment:
this probably doesn't need the defaults but I guess it's okay

(is there a good julia way to "forward" the fact that the argument is missing, apart form `Missing` ?)

Reply:
Oh, yeah, this internal function doesn't really need a default, I'm going to remove it.

> (is there a good julia way to "forward" the fact that the argument is missing, apart form Missing ?)

I guess the julian way would be a different method.